sdk/core/core/test/cmocka/test_az_credential_client_secret.c

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "az_credentials.h"
+#include "az_test_definitions.h"
+#include <az_credentials_internal.h>
+#include <az_http.h>
+#include <az_http_internal.h>
+#include <az_span.h>
+
+#include <stddef.h>
+
+#include <setjmp.h>
+#include <stdarg.h>
+
+#include <cmocka.h>
+
+#include <_az_cfg.h>
+
+static void test_credential_client_secret(void** state)
+{
+  (void)state;
+
+  az_credential_client_secret credential = { 0 };
+  // init credential_credentials struc
```

Comment:
remove or fix comment?

Reply:
Missed it after copying-pasting. I'll remove it. And I will update in the place where I copied it from.

Reply:
Done.